scripts/merge-gisaid-owid.py

```diff
@@ -31,6 +31,9 @@
 parser.add_argument('--gisaid-metadata-file', default='', help='GISAID metadata file export path')
 parser.add_argument('--merged-gisaid-owid-out', default='',
     help='Path where to write merged GISAID/OWID CSV')
+parser.add_argument('--daily-only', action='store_true', default=False,
```

Comment:
just so we're not confused later, may be worth saying that the weekly file will be created regardless as part of the help string

Reply:
switched this for clarity to a -make-weekly-file flag that's True if present and False if omitted